src/noteburst/worker/nublado.py

```diff
@@ -86,10 +87,36 @@ async def spawn(
         )
 
         await nublado_client.auth_to_hub()
-        await nublado_client.spawn_lab(config=nublado_image)
+        logger.info("Auth'd to hub.")
+
+        # If we previously shut down uncleanly, our lab may still be running
+        await nublado_client.stop_lab()
+        logger.info("Stopped lab.")
```

Comment:
Maybe drop this log message. It could be confusing because we don't actually know if we had to stop a Jupyterlab pod. I assume `stop_lab` is just a fire-and-forget with no confirmation or feedback.